assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -283,6 +340,40 @@ const baseSelectors = {
 
 		return dismissedTourSlugs.includes( slug );
 	} ),
+
+	/**
+	 * Gets the timestamp for the last dismissed feature tour.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {number} Timestamp of the last dismissal.
+	 */
+	getLastDismissedAt( state ) {
+		return state.lastDismissedAt;
+	},
+
+	/**
+	 * Determines whether feature tours are on cooldown (i.e. the last
+	 * dismissal was within the cooldown time span).
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {undefined|boolean} Whether feature tours are on cooldown or undefined.
+	 */
+	areFeatureToursOnCooldown: ( createRegistrySelector( ( select ) => () => {
```

Comment:
There's an extra wrapping `()` around the `createRegistrySelector` here which isn't needed.